### PR TITLE
Added content filter members to ChatCompletions and throw errors when content is filtered.

### PR DESCRIPTION
**Pull request by robgruen**
Added content filter members to ChatCompletions and throw errors when content is filtered.

### CODE REVIEW

File: ts/packages/aiclient/test/testVerifyContentFilterCheck.spec.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+describe("this is a test", () => {
+    describe("testVerifyContentFilterCheck", () => {
+        it("VerifyContentFilter", async () => {
+            //let model = createChatModel(apiSettingsFromEnv(ModelType.Embedding));
+
+            //const result = await model.complete("what is 2+2?");
+
+            //expect(result).toContain("4");
+            expect(true).toEqual(true);
```
Comment (curtisman) [inline ts/packages/aiclient/test/testVerifyContentFilterCheck.spec.ts]: what is this for?

Comment (robgruen) [reply ts/packages/aiclient/test/testVerifyContentFilterCheck.spec.ts]: That wasn't supposed to be checked in yet....was writing a test for the verifyContentIsSafe() method.

Comment (robgruen) [reply ts/packages/aiclient/test/testVerifyContentFilterCheck.spec.ts]: I'll finish the test before I merge the code but wanted you all to get a chance to look at the other stuff.

Comment (robgruen) [reply ts/packages/aiclient/test/testVerifyContentFilterCheck.spec.ts]: I'll actually remove the test, finish the PR, then finish the test.

**Pull request merged**